grammars/bucklescript-hover-type.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This is for a specific editor. No need to include this

Reply:
Done.